gql-inspector-checker.js

```diff
@@ -70,6 +69,8 @@ module.exports = (props) => {
         change.message =
           'New arguments must include a description with a version number in the format "Added in XX.XX.X.", ' +
           change.message;
+      } else {
+        change.criticality.level = "WARNING";
```

Comment:
Please refer to [the criticality level](https://github.com/kamilkisiela/graphql-inspector/blob/master/packages/core/src/diff/changes/change.ts#L1-L5). It's better to change to "DANGEROUS".